ThatsPrettyClever.cs

```diff
@@ -18,6 +18,13 @@ public partial class ThatsPrettyClever : Form
         private int _rollNum;
         private DiceCollection _diceCap;
 
+        private Game CurrGame = Game.ThatsPrettClever;
+        private enum Game
+        {
+            ThatsPrettClever,
```

Comment:
You're not really saving anything by not including the "y" in "Pretty" - the keys are like right next to each other, haha.